### DISCUSSION TITLE
Hi All I m really new to this stuff so [Jun 04]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2329] [Discord Threads]
> 🙋‍♂️ **treb0r**

**Comment by o1lab** [comment]
> 👨‍💻 **pranavxc**  
#### Both are  Independent application... if you made any db table alteration from Hasura it won't reflect in NocoDB  
#  
> 💁 **o1lab**  
#### to answer : yes - you can use it like that   
#### to identify schema changes - you can do sync schema  
#  
> 💁 **o1lab**  
#### https://docs.nocodb.com/setup-and-usages/sync-schema  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/982053843565166652)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
